src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/multinomial.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 Intel Corporation
```

Comment:
Please move this file to shared [folder](https://github.com/openvinotoolkit/openvino/tree/master/src/tests/functional/shared_test_classes/include/shared_test_classes/single_op).

Reply:
Done!